src/client/serialize/daos_serialize.c

```diff
@@ -261,6 +340,12 @@ daos_cont_serialize_props(hid_t file_id, daos_prop_t *prop_query)
 			if (rc != 0) {
 				D_GOTO(out, rc);
 			}
+		} else if (type == DAOS_PROP_CO_ROOTS) {
+			entry = &prop_query->dpp_entries[i];
+			rc = serialize_roots(file_id, entry, prop_str);
+			if (rc != 0) {
+                		D_GOTO(out, rc);
```

Comment:
(style)  code indent should use tabs where possible
(style)  please, no space before tabs